libfabric.spec

```diff
@@ -13,7 +13,8 @@ License: GPLv2 or BSD
 %endif
 Url: https://www.github.com/ofiwg/libfabric
 Source: https://github.com/ofiwg/%{name}/archive/v%{version}.tar.gz
-Patch0: https://github.com/ofiwg/libfabric/compare/v1.9.0..62f6c937601776dac8a1f97c8bb1b1a6acfbc3c0.patch
+Patch0: https://github.com/ofiwg/libfabric/compare/v1.9.0..8fa7c5bbbfee7df5194b65d9294929a893eb4093.patch
+Patch1: https://github.com/daos-stack/libfabric/compare/sockets_provider.patch
```

Comment:
This doesn't seem to be a valid URL.  Is this patch in a git repo somewhere or just a flat patch file at this time?

Reply:
I am trying to upload the patch as part of this PR. How should i point to that patch?

Reply:
> I am trying to upload the patch as part of this PR. How should i point to that patch?

In fact, in RPM spec files, patches are usually just specified as unqualified file names such as:

```
Patch1: sockets_provider.patch
```

I only used a URL for `Patch0` as it is _fetchable_ from somewhere and using the URL better documents it's origin.

Will you be also including this patch as an _scons_local_ patch then?

Reply:
As discussed over PM, once this PR lands, CART's version will specify url to this patch as we now support specifying remote patches via [patch_versions] section in build.config